schema.json

```diff
@@ -0,0 +1,226 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://alpheios.net/schemas/apis/user-word/item.json",
+  "type": "object",
+  "title": "Schema for a User Word Item",
+  "definitions": {
+    "targetWord": {
+      "$id": "#/definitions/targetWord",
+      "type": "string",
+      "title": "Target Word Schema",
+      "examples": [
+        "cespitum"
+      ],
+      "pattern": "^(.+)$"
+    },
+    "languageCode": {
+      "$id": "#/definitions/languageCode",
+      "type": "string",
+      "title": "Language Code Schema",
+      "examples": [
+        "lat"
+      ],
+      "pattern": "^\\w\\w\\w$"
```

Comment:
It seems to be a pattern for "\w\w\w". I'm wondering if this is correct

Reply:
It's a test for 3 word characters. That is correct (although it could be more precise)

Reply:
Oh, I think I've missed that the backslashes are escaped in JSON, and "^\\w\\w\\w$" would be converted into `/\w\w\w/`. My bad!